packages/theme-check-docs-updater/src/themeLiquidDocsManager.ts

```diff
@@ -23,16 +23,16 @@ function dataRoot() {
   }
 }
 
-async function fallback<T>(name: string, defaultValue: T): Promise<T> {
+async function fallback<T>(name: string, defaultValue: T, shouldParse = true): Promise<T> {
   try {
     const content = await fs.readFile(path.resolve(dataRoot(), name), 'utf8');
-    return JSON.parse(content);
+    return shouldParse ? JSON.parse(content) : content;
```

Comment:
I'm still not happy with this. It's very awkward to use and not composable. Should probably make the chaining kind of different.